python/cudf_polars/cudf_polars/containers/column.py

```diff
@@ -33,10 +34,32 @@ def __init__(
         null_order: plc.types.NullOrder = plc.types.NullOrder.BEFORE,
     ):
         self.obj = column
+        self.is_scalar = self.obj.size() == 1
+        if self.is_scalar:
+            is_sorted = plc.types.Sorted.YES
```

Comment:
sortedness is also `Yes` for length-0, so perhaps:
```suggestion
        if self.obj.size() <= 1:
            is_sorted = plc.types.Sorted.YES
```